src/components/InstalledPage.jsx

```diff
@@ -14,6 +14,19 @@ import snarkdown from 'snarkdown'
 import WithCozyIcon from 'components/WithCozyIcon'
 import platforms from 'platforms'
 
+const PlatformButton = props => {
+  const { icon, ...rest } = props
+  const Component = rest.href ? ButtonLink : Button
```

Comment:
This feels a bit weird ? Why not using always a ButtonLink here ?

Reply:
Because that's a mistake, I should alway use ButtonLink :) thanks